client/src/browser_action/popup.js

```diff
@@ -17570,202 +17570,122 @@ module.exports = getIteratorFn;
 
 "use strict";
```

Comment:
I don't think popup.js should every be committed, since this is a transpiled file. I believe David has add it to .gitignore in chat branch.